client/app/components/Input/Input.scss

```diff
@@ -151,7 +151,9 @@
     text-transform: capitalize;
   }
 }
-
+.clr-white {
```

Comment:
This is an unnecessary class. Why not just add `color: $white;` to an existing element class?

Reply:
Adding `color: $white;` to the existing `checkboxLabel` class was not taking effect because we have set the [default color with element selector specificity](https://github.com/ifmeorg/ifme/blob/d9e34cdb86b56cbabc2dd4ffd86108e4f96c65b7/app/assets/stylesheets/dashboard/dashboard_section.scss#L131). So that's why I created a separate class to override the label's color.

Reply:
Acknowledged.